ruma-events/src/enums.rs

```diff
@@ -116,25 +121,25 @@ pub type AnyStrippedStateEventStub = StrippedStateEventStub<AnyStateEventContent
 pub type AnyToDeviceEvent = ToDeviceEvent<AnyToDeviceEventContent>;
 
 /// Any event.
-#[derive(Clone, Debug, Deserialize, Serialize)]
+#[derive(Clone, Debug, Serialize)]
 #[serde(untagged)]
 pub enum AnyEvent {
     /// Any basic event.
     Basic(AnyBasicEvent),
-    /// `"m.presence"`, the only non-room event with a `sender` field.
-    Presence(PresenceEvent),
     /// Any ephemeral room event.
     Ephemeral(AnyEphemeralRoomEvent),
     /// Any message event.
     Message(AnyMessageEvent),
+    /// `"m.presence"`, the only non-room event with a `sender` field.
```

Comment:
Why did you move this variant?

Reply:
The others seemed to be in alphabetical order except this one?

Reply:
The almost alphabetical order is pure coincidence. I ordered them the same way they are declared - simplest one first, most complex one last.